.gitignore

```diff
@@ -17,6 +17,7 @@ out
 build-local.properties
 .gradle
 build
```

Comment:
Possibly, not sure.  It was in every directory though so I added it to .gitignore.